### ISSUE TITLE
The problem with scrolling

### ISSUE THREAD

**Issue by Gorfar** [vdrhtc/concrypt#7]
When I scroll a message history next situation can be occured: In the "cache" there isn't more messages loaded and Concrypt want to load it, he wait for a instant and after loading scroll to the start of this loaded message group. I propose to add a time constant that will enhance scrolling continuity.

**Comment by vdrhtc** [comment]
You mean to make a smooth transition-animation to the beginning of the next group?

**Comment by Gorfar** [comment]
Yes and no. A smooth transition-animation not to the beginning, but to the end. 
The problem is that if i want to find smth in the conversation I scroll and all conversation is jumping.
